src/common/getType.ts

```diff
@@ -579,11 +631,9 @@ function getOpCodeType(
         ["T1", (x) => x[0]],
         ["T2", (x) => x[1]]
       )[1];
+    case null:
+      throw new Error("Cannot determine type based on null opcode.");
```

Comment:
Would be good to clarify (in the error message, perhaps) what is a null opcode. Does it represent that a backend transformation has created a language-specific node?